internal/stores/sql.go

```diff
@@ -34,7 +34,7 @@ func NewEphemeralSQLiteConnection(name string) gorm.Dialector {
 
 // NewSQLiteConnection opens a sqlite db at the given path.
 func NewSQLiteConnection(path string) gorm.Dialector {
-	return sqlite.Open(path)
+	return sqlite.Open(fmt.Sprintf("file:%s?_busy_timeout=5000", path))
```

Comment:
use a `PRAGMA` to set the `busy_timeout` - that way we are consistent with the FK checks